src/components/Board/Symbol/Symbol.js

```diff
@@ -38,35 +78,17 @@ function Symbol(props) {
     ...other
   } = props;
   const [src, setSrc] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(
     () => {
-      async function getSrc() {
-        let image = null;
-        if (keyPath) {
-          const arasaacDB = await getArasaacDB();
```

Comment:
Please avoid this kind of refactoring when it is not strictly necessary. It confuses the review.
Also, in this case, the function was moved from it scope.